sdk/typescript/packages/sdk/src/mixnet/wasm/subscriptions.ts

```diff
@@ -0,0 +1,69 @@
+import type { EventHandlerFn } from './types';
+import { EventKinds } from './types';
+
+type ISubscriptions = {
+  [key: string]: Array<EventHandlerFn<unknown>>;
```

Comment:
```suggestion
 [key in EventKinds]?: Array<EventHandlerFn<unknown>>;
```
This would tighten the key type a little